### ISSUE TITLE
Korriban Dreshdae - Various issues

### ISSUE THREAD

**Issue by DarthParametric** [KOTORCommunityPatches/K1_Community_Patch#160]
- [x] One of the Sith hopefuls in the entrance cutscene is a Twi'lek female is in stripper gear. There's another as a random background NPC outside the cantina and several inside the Czerka office. They should switch to the commoner clothes appearance rows. 
- [x] In the run-in with Lashowe, her lackeys turn to face her. There's a background mook patrolling that can freak out party members causing them to constantly reposition. It's a triggered convo so the party can be jumped into place. 
- [x] In the cantina there are two pilots that use different heads but both are wearing green clothes. 
- [ ] ~In the cantina there are two patrons to the left as you enter that are oddly positioned, one in front of the other.~ On reflection I think this is probably fine.
- [x] ~Yuthura wears Sith officer gear, but different from in the Academy. Looks like a darker colour.~ Lashowe as well. Edit: Another one for @JCarter426 's Back in Black partial integration. Although I guess we are keeping Yuthura in the dark outfit.
- [x] Yuthura's conversation needs the party to be jumped into place
- [x] When talking to Yuthura, the interjection by one of the "slaves" is done facing the player, not Yuthura

**Comment by ebmar** [comment]
> In the run-in with Lashowe, her lackeys turn to face her... It's a triggered convo so the party can be jumped into place.

Yeah they should! The way I see it too; the default trigger seem to be too far away from an ideal so it's kind of weird to seem them talking from afar. Anyway, apparently there're redundant -if not, breaking- scripts inside `kor33_lashowe.dlg`; that initial convo with Lashowe and her mofos in Dreshdae `korr_m33aa`. They're -
```
k_pkor_lashturn1.ncs
k_pkor_lashturn2.ncs
```
Breaking - because with those scripts fired tend to break the facing of participants. These are some entries that uses them -
```
Entry 29
Entry 32
Entry 35
Entry 39
```
With them removed now Speaker-Listener tag for the aforementioned entries will not be ignored.

**Comment by DarthParametric** [comment]
1.10.0 beta test 2 observations:

- [ ] When first meeting Yuthura in the cantina, she doesn't face the PC on the first line, presumably due to the party jump.
